src/focus.service.ts

```diff
@@ -406,19 +406,27 @@ export class FocusService {
       }
     }
 
-    // Otherwise see if we can handle it...
+    return false;
+  }
+
+  public defaultFires(ev: ArcEvent, scrollSpeed: number = Infinity): boolean {
```

Comment:
It is already optional by having a default value, typescript will not allow you to also put a `?` there